src/useMultiRootEditor.tsx

```diff
@@ -505,7 +505,7 @@ const useMultiRootEditor = ( props: MultiRootHookProps ): MultiRootHookReturns =
 				data || /* istanbul ignore next -- @preserve: It should never happen, data should be always filled. */ {}
 			);
 
-			const hasModifiedData = dataKeys.some( rootName =>
+			const hasModifiedData = dataKeys.filter( rootName =>
```

Comment:
Please rename this variable as it is no longer a boolean, but an array of strings.

Reply:
Fixed in f09915b7593bdbb57cca4e04221b2088fb69e43f.